BookClub/BookClub.xproj

```diff
@@ -15,5 +15,9 @@
   <PropertyGroup>
```

Comment:
Totally agree with this. Every project must have its own scope and its own reason to be, that way we can have logically separated components that are responsible of a single area of the application.